.github/workflows/ci.yml

```diff
@@ -25,7 +25,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        rust: [1.70.0, nightly]
+        rust: [1.71.0, nightly]

```

Comment:
I like to use the same Rust version for all CI jobs, but I can change that later.

Reply:
28e83a9